toolbox/cluster.py

```diff
@@ -24,14 +24,16 @@ def set_scale(instance_type, scale, force=False):
         Args:
             instance_type: The instance type to use, for example, g4dn.xlarge
             scale: The number of required nodes with given instance type
+            base_machineset: Name of a machineset to use to derive the new one. Defaults: pickup the first machineset found in `oc get machinesets -n openshift-machine-api`.
         """
         opts = {
                 "machineset_instance_type": instance_type,
                 "scale": scale,
+                "base_machineset": base_machineset,
             }
 
         if force:
-            opts = {**opts, "force_scale": "true"}
+            opts["force_scale"] = "true"
```

Comment:
Sometimes I do things in ridiculous ways when I forget there are absolutely trivial ways to do them :sweat_smile:

Reply:
🙃 😉